### ISSUE TITLE
[BUG] <New instance in a different folder - options.txt is not created>

### ISSUE THREAD

**Issue by endervad** [PojavLauncherTeam/PojavLauncher#3623]
### Describe the bug

I noticed that by default if there's no `options.txt` in the default `.minecraft` folder, this file is created with the following contents:
```
overrideWidth:2560
fullscreen:off
overrideHeight:1600
```
However if you try creating a new instance in a different folder and starting it, this file is not created and therefore resolution is not overridden. Depending on the version of Minecraft, this either doesn't matter (newer versions) or game's resolution becomes weird (older versions). I didn't test this extensively but it does occur on vanilla 1.6.4.
Since pre-creating `options.txt` already exists, it's only right to have it pre-created in the instances too.

### The log file and images/videos

Notice the cursor position which represents where the highlighted button "actually" is.
![Screenshot_20221031-085420_PojavLauncher (Minecraft_ Java Edition pour Android)](https://user-images.githubusercontent.com/10913278/198940959-e3845213-2194-4691-a241-edd5a3ff4226.png)
![Screenshot_20221031-085441_PojavLauncher (Minecraft_ Java Edition pour Android)](https://user-images.githubusercontent.com/10913278/198940974-bcae1719-ee26-4293-bdb0-e807e6816c73.png)
I toggled this one out of curiosity - the resolution became fine but buttons' hitboxes went out of bounds and are like x2 larger than the actual resolution.
![Screenshot_20221031-085506_PojavLauncher (Minecraft_ Java Edition pour Android)](https://user-images.githubusercontent.com/10913278/198940984-28a549a1-dd51-4d36-b80d-f12fe863aea2.png)

### Steps To Reproduce

```markdown
Normal behavior:
1. Delete .minecraft/options.txt
2. Launch vanilla 1.19.2
3. Quit w/o going into the settings
There will be options.txt with just the resolutions lines.
4. Delete .minecraft/options.txt again
5. Launch vanilla 1.6.4
6. Quit w/o going into the settings
Again, there's options.txt but with all sorts of settings. Resolution overrides are in place too.

Current behavior in instances
1.19.2:
1. Create new vanilla 1.19.2 in a new non-existent folder
2. Launch this instance
3. Quit w/o going into the settings
There will be no options.txt - Minecraft didn't create it, neither did the launcher. For me the actual resolution was fine though.
1.6.4:
1. Create new vanilla 1.6.4 in a new non-existent folder
2. Launch this instance
In my case the resolution was broken: the image was 1/6 size of my screen in the bottom-left corner while the buttons' hitboxes were in the upper-left corner of the same size. I found that out by moving virtual cursor.
```

### Expected Behavior

`options.txt` should be created for every new instance the same way as how `options.txt` is pre-created for the default `.minecraft` folder.

### Platform

```markdown
- Device model: Lenovo Legion Y700
- CPU architecture: aarch64
- Android version: 12
- PojavLauncher version: dahlia-107-b6f15cb42-v3_openjdk
```

### Anything else?

Sorry if I went a bit too extensive in such a seemingly small bug lol

**Comment by GaucovyAero** [comment]
well, the first issue is that you weren't fullscreen, the second issue is that you had a custom directory set, it causes the screen to go brr kinda

**Comment by endervad** [comment]
I'm pretty sure that I described only one issue which you described as a second one. Yes, resolution is not okay when custom directory is set, however it's okay when there's no custom directory, i.e. `.minecraft` folder is used. And I've just digged out the reason behind this in the first post: in `.minecraft` folder `options.txt` is created beforehand, however in custom folders it's not. So the fix would be to apply the same logic of creating `options.txt` to the custom directories.
